lib/add_magic_comment.rb

```diff
@@ -3,14 +3,15 @@
 # A simple library to prepend magic comments to multiple ".rb" files
 
 module AddMagicComment
-  MAGIC_COMMENT_PREFIX  = "frozen_string_literal"
-  MAGIC_COMMENT_PATTERN = /^-?# *#{MAGIC_COMMENT_PREFIX}/
+  MAGIC_COMMENT_PREFIX  = 'frozen_string_literal'
```

Comment:
It's a stylistic preference, mostly (e.g., RubyMine flags double-quoted literal strings not containing interpolation), but it's actually faster for Ruby to parse, because it's not possible for there to be anything dynamic going on inside the string.  For a similar reason, it's easier for humans to parse as a non-dynamic string.